__tests__/app.test.js

```diff
@@ -64,4 +64,18 @@ describe("GET /api/articles/:article_id", () => {
                 expect(body.msg).toBe("article not found")
             }) 
     })
+})
+
+describe.only("PATCH /api/articles/:article_id", () => {
+    test("responds with the updated article", () => {
+        const articleUpdate = { inc_votes: 5 };
+        return request(app)
+            .patch("/api/articles/1")
+            .send(articleUpdate)
+            .expect(200)
+            .then((res) => {
+                const article = res.body.article;
+                expect(article.votes).toBe(105);
```

Comment:
Might be worth asserting against the `article_id` also to make sure that article is the one you are updating!